CHANGELOG

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+1.0.1 (2017-01-07)
```

Comment:
I'd rather have this:

```
master (unreleased)
-------------------
```

you never know when the next release would happen ;o)